cluster/images/conformance/go-runner/BUILD

```diff
@@ -41,6 +41,7 @@ go_test(
         "env_test.go",
         "tar_test.go",
     ],
+    data = glob(["testdata/**"]),
```

Comment:
revert this?

Reply:
hack/update-bazel.sh created it - I wasn't quite sure why. Just removed it.